synapse/types.py

```diff
@@ -395,8 +396,30 @@ class RoomStreamToken:
 
     Live tokens start with an "s" followed by the "stream_ordering" id of the
     event it comes after. Historic tokens start with a "t" followed by the
-    "topological_ordering" id of the event it comes after, followed by "-",
+    "topological_ordering" id of the event it comes after, followed by "~",
```

Comment:
Woops, this hasn't changed at all. I used to use `-` for the vector clock token but decided to change it (to avoid woes with negative numbers) and clearly got a bit too enthusiastic with updating the do